xcmailr-webapp/src/main/java/models/PageList.java

```diff
@@ -119,7 +118,7 @@ public int getPageCount()
             return 1;
         }
 
-        int entrys = allEntrys.size();
+        int entrys = entries.size();
```

Comment:
Please rename `entrys` as well. Reading this hurts. ;-)

Reply:
Done.